src/sql/src/func.rs

```diff
@@ -2142,8 +2148,8 @@ pub static PG_CATALOG_BUILTINS: Lazy<BTreeMap<&'static str, Func>> = Lazy::new(|
             params!(RangeAny) => UnaryFunc::RangeLowerInf(func::RangeLowerInf) => Bool, 3853;
         },
         "lpad" => Scalar {
-            params!(String, Int64) => VariadicFunc::PadLeading => String, 879;
-            params!(String, Int64, String) => VariadicFunc::PadLeading => String, 873;
+            params!(String, Int32) => VariadicFunc::PadLeading => String, 879;
+            params!(String, Int32, String) => VariadicFunc::PadLeading => String, 873;
```

Comment:
In general yes, it is backwards incompatible. It's possible that someone had a view that used one of these functions with a number out of range of `Int32`. Now their view would fail to plan causing their environmentd to crash when trying to boot up.

Reply:
I think the boot problem would occur if the input type can't be coerced/cast into an int32 (but somehow could be cast into an int64?). I'll do some experiments to see what kinds of things would be broken by this. From there we can decide if that's ok, or if the fixes should leave the argument types and fix in a different way.

Reply:
I agree.